site/content/contribute/getting-started/code-reviews.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Code Reviews"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+All changes to the product must be reviewed.
+
+* User experience changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+* Code changes must be reviewed by at least two [core committers](http://localhost:1313/contribute/getting-started/core-committers/#core-committers).
+* Documentation changes should be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers) and at least one [core committer](/contribute/getting-started/core-committers/#core-committers).
+
+If you are not a core committer
+-------------------------------
+
+1. Submit your pull request.
+2. Reach out to the author of the ticket your PR resolves.
+    * Typically this means [mentioning](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#mentioning-people-and-teams) their username as a comment on the pull request.
+    * Product managers and core committers are often on the lookout for new PRs, and may handle this for you automatically.
+    * When in doubt, ask for help in the [Developers](https://community.mattermost.com/core/channels/developers) channel on our community server.
+3. Wait for a review.
+    * The ticket author, another core committer or a product manager will assign reviewers and label your pull request appropriately.
+    * Expect some interaction with at least one reviewer within 5 business days.
```

Comment:
Can they at-mention the ticket author, or the acting community manager (Hanzei + Jason) if no interaction within 5 business days